x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_inventory/service_inventory.spec.ts

```diff
@@ -82,10 +80,13 @@ describe('When navigating to the service inventory', () => {
       apiRequestsToIntercept.map(({ endpoint, aliasName }) => {
         cy.intercept('GET', endpoint).as(aliasName);
       });
+
+      cy.loginAsReadOnlyUser();
+      cy.visit(serviceInventoryHref);
     });
 
     it('with the correct environment when changing the environment', () => {
-      cy.wait(aliasNames);
+      cy.wait(aliasNames, { requestTimeout: 10000 });
```

Comment:
Even with `10000` they fail sometimes

Reply:
hmm, is there any way we can figure it out? maybe add & log some request timings for the time being?

Reply:
I've increased the `requestTimeout` to 30s and temporarily removed cypress retries in https://github.com/elastic/kibana/pull/129475/commits/761dc8ede42981e23560751c708a410971fd57f2 and in [50 test runs](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/398) it doesn't seem to have failed

Reply:
Actually one is still failing :|